src/styles/blocks/header.scss

```diff
@@ -0,0 +1,21 @@
+.header,
+.nav__list {
```

Comment:
The '.nav__list' styles are declared in the 'header.scss' file. It's a good practice to keep the styles for different blocks in separate files. Consider moving '.nav__list' styles to 'nav.scss'.